src/applications/find-forms/components/FindVaForms.jsx

```diff
@@ -42,11 +42,11 @@ export default () => (
         </p>
         <a
           className="vads-u-display--block vads-u-padding-top--1 vads-u-text-decoration--none"
-          href="/disability/how-to-file-claim/"
+          href="/disability/file-disability-claim-form-21-526ez/"
           onClick={onFeaturedContentClick('File a VA disability claim')}
         >
           <span>
-            Read more
+            Apply Online
```

Comment:
I think this should be `Apply online` to follow our sentence case pattern, right?

Reply:
Haha whoops. That would a nice catch.

Thank you!